.github/workflows/integration-md.yml

```diff
@@ -18,22 +18,22 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [net6.0]
+        version: [net7.0]
```

Comment:
My gut says this is fine. In the before world, there were more significant differences between versions of .NET. The OTLP exporter integration test exercised the exporter using the two different gRPC libraries. This difference does not exist between net6 and net7.

Reply:
Also +1 to keeping the CI short!

Reply:
We can look at putting it back. My main goal right now is just to unblock VS.